tests/load-tests/pkg/journey/handle_collections.go

```diff
@@ -1,13 +1,16 @@
 package journey
 
-import "fmt"
```

Comment:
Hello @rh-rahulshetty . Totally not important, but could you please leave these logging statements as is? I thried to split them to groups like "standard library", "out code" and "third party". No strong feeling here, but if you want to change it, let's do it in separate pr for all the files.

Reply:
Hello Jan,
I guess the import statements grouping got updated due to IDE's auto-format, I'll revert back to the old grouping.